# 4.8 Twitter users and news

In [3]:
import math
""" Confidence Interval of 95%  Function

pe = the population point estimate
std = standard deviation of the population
sample_size = size of the sample

"""
#Standard error method that comes into computation
def standard_error(std, sample_size):
    return (std/(math.sqrt(sample_size)))


def ci_99(pe, se):
    z = 2.576

    ci_minus = pe - (z * se)
    ci_plus = pe + (z * se)
    ci = (ci_minus,ci_plus)
    #returns a tuple
    return ci

**Given Data :** 
<br/>
point estimate = 52%<br/>
Standard Error = 2.4%

Creating a 99 % confidence interval for the proportion of US adults Twitter users who get some news on Twitter.

In [4]:
ci_99(52,2.4)

(45.8176, 58.1824)

**<u>Inference : </u> : ** We are 99% confident that the proporion of U.S. adults using Twitter getting some news out of it falls between 45.8176, 58.1824